### ISSUE TITLE
DeepSSM Use Case  - adapt to include full pipeline

### ISSUE THREAD

**Issue by jadie1** [SCIInstitute/ShapeWorks#1409]
We need to adapt the DeepSSM use case to include the full pipeline:

- Define a train/test split
- Groom the train and test images using shape
- Optimize shapeworks on train set only
- Augment data using train set
- Split train into train and validation sets and train the model
- Test the model on the test set - use surface to surface distance to analyze
